.dev/compose.backbone.env

```diff
@@ -1 +1 @@
-BACKBONE_VERSION=6.6.1
+BACKBONE_VERSION=6.7.2
```

Comment:
if we bump we should bump properly: [`6.10.0`](https://github.com/nmshd/backbone/releases/tag/6.10.0) is out